WrightTools/collection/_collection.py

```diff
@@ -102,6 +102,17 @@ def create_collection(self, name="collection", position=None, **kwargs):
         setattr(self, name, collection)
         return collection
 
+    def convert(self, units):
+        """Convert units of a collection.
+        Parameters
+        ----------
+        units: string
+            Units to convert to.
+        """
+
+        for data in self.data:
```

Comment:
Collection objects have no attribute `data`, I believe you can just do `for i in self:` or `for i in self.items():`